src/index.tsx

```diff
@@ -34,6 +35,11 @@ import './index.css';
 function Index() {
   const user = useSelector((state: RootState) => state.user.currentUser);
 
+  const noChecking = async (): Promise<TypePromise<string>> =>
```

Comment:
Maybe we should keep allowUsers as an optional argument in loaderWrapper. That way, if no argument is passed, then it won't have to check, so that we can remove this function